p4app-exercises/p4runtime.p4app/README.md

```diff
@@ -1,174 +1,183 @@
-# Implementing Basic Forwarding
+# Implementing a Control Plane using P4Runtime
 
 ## Introduction
 
-The objective of this exercise is to write a P4 program that
-implements basic forwarding. To keep things simple, we will just
-implement forwarding for IPv4.
-
-With IPv4 forwarding, the switch must perform the following actions
-for every packet: (i) update the source and destination MAC addresses,
-(ii) decrement the time-to-live (TTL) in the IP header, and (iii)
-forward the packet out the appropriate port.
+In this exercise, we will be using P4Runtime to send flow entries to the 
+switch instead of using the switch's CLI. We will be building on the same P4
```

Comment:
isn't that confusing? None of the p4app-based tutorial exercises use the CLI. They all use P4Runtime.

Reply:
I started with this README.md by copying the one from exercises/p4runtime/README.md.  I suspect that in the exercises directory, if you go in the same order of basic, basic_tunnel, then p4runtime, as recomended, then this is probably the first one in that sequence that uses P4Runtime.

I can update that comment in this README.md, since in p4app-exercises, all of basic, basic_tunnel, and p4runtime use P4Runtime.